docs/decimal128.md

```diff
@@ -49,11 +88,28 @@ OrderDecimal.find({where: {count: '0.0005'}}, function(err, orders) {
 
 The connector automatically converts the condition from string to decimal.
 
+When query nested properties, you still need to do the conversion yourself, like
+
+```js
+// query decimal property in a nested array
+const arrCond = {where: {lines: {elemMatch: {unitPrice: Decimal128.fromString('0.0005')}}}};
```

Comment:
I have a question. How can LB users query nested Decimal128 properties via the REST API?

IIUC, the following query will not work because there is no code to automatically convert "0.0005" string into Decimal128 instance.

```ts
GET /orderdecimals?filter[where][lines][elemMatch][unitPrice]=0.0005
```

Is this something we should worry about and improve? (Possibly in a follow-up pull-request.)

Reply:
@bajtos yes. Detecting the decimal property in a nested query is more complicated than in a model instance, since a query could contain extended operators. I would suggest customer use some workaround for the query...at least give the feature a lower severity level than sev1 to give us more time plan for it.

Reply:
@jannyHou Makes sense. Could you please open a follow-up GH issue for this?